commands/images/reddit.js

```diff
@@ -0,0 +1,75 @@
+const { MessageEmbed, MessageActionRow, MessageButton, Guild, GuildChannel, Channel, Message } = require('discord.js');
+const got = require("got")
+module.exports = {
+	name: 'reddit',
+	description: 'Shows images from a subreddit.',
+	usage: 'reddit <search query>',
+	cooldown: '10',
+	args: true,
+	aliases: ['red', 'reddit'],
+	async execute(client, message, args, functions) {
+		let subName = args.at(0);
+		console.log(args, args.length);
+		if (args.length > 1) {
```

Comment:
please add a check to see if the specified subreddit even exists/if the url even has any content.